diff -ruN e3sm_diags-2.6.0/e3sm_diags/plot/cartopy/arm_diags_plot.py e3sm_diags-2.6.0-patch/e3sm_diags/plot/cartopy/arm_diags_plot.py
--- e3sm_diags-2.6.0/e3sm_diags/plot/cartopy/arm_diags_plot.py	2021-11-02 11:02:13.000000000 -0700
+++ e3sm_diags-2.6.0-patch/e3sm_diags/plot/cartopy/arm_diags_plot.py	2022-01-18 19:23:06.735543687 -0700
@@ -160,7 +160,6 @@
         ax1.errorbar(
             bin_center,
             pr_binned_mean,
-            xerr=0,
             yerr=errorbar_precip.squeeze(),
             ls="none",
             color="black",
@@ -193,7 +192,6 @@
         ax2.errorbar(
             bin_center,
             pr_probability,
-            xerr=0,
             yerr=errorbar_precip_binom.T,
             fmt="none",
             color="black",
@@ -249,7 +247,7 @@
         freq_precipitating_points[freq_precipitating_points == 0] = np.nan
         freq_cwv[freq_cwv == 0] = np.nan
 
-        # ax3.errorbar(bin_center, freq_precipitating_points, xerr=0, yerr=errorbar_precip_points.squeeze(), ls='none', color='black')
+        # ax3.errorbar(bin_center, freq_precipitating_points, yerr=errorbar_precip_points.squeeze(), ls='none', color='black')
         ax3.scatter(
             bin_center,
             freq_cwv,
