diff --git a/tests/system/test_diags.py b/tests/system/test_diags.py
index 9c37d93d..d2b172aa 100644
--- a/tests/system/test_diags.py
+++ b/tests/system/test_diags.py
@@ -143,7 +143,8 @@ class TestAllSets(unittest.TestCase):
                 TestAllSets.results_dir,  # type: ignore
             )
 
-    def check_html_image(self, html_path, png_path, full_png_path):
+    # Set `check_images=True` to do a pixel-by-pixel image comparison check.
+    def check_html_image(self, html_path, png_path, full_png_path, check_images=False):
         img_src = None
         option_value = None
         href = None
@@ -169,15 +170,16 @@ class TestAllSets(unittest.TestCase):
         path_to_actual_png = full_png_path
         path_to_expected_png = "../unit_test_images/{}".format(png_path)
 
-        mismatched_images = []  # type: ignore
-        compare_images(
-            self,
-            mismatched_images,
-            image_name,
-            path_to_actual_png,
-            path_to_expected_png,
-        )
-        self.assertEqual(mismatched_images, [])
+        if check_images:
+            mismatched_images = []  # type: ignore
+            compare_images(
+                self,
+                mismatched_images,
+                image_name,
+                path_to_actual_png,
+                path_to_expected_png,
+            )
+            self.assertEqual(mismatched_images, [])
 
     def check_plots_generic(
         self, set_name, case_id, ref_name, variables, region, plev=None
